Makefile

```
@@ -0,0 +1,10 @@
+build_proto:
```

When using make targets to do actions, use `.PHONY` to ensure the target always runs, even if a file exists with the same name as the target.

```suggestion
.PHONY: build_proto
build_proto:
```